### ISSUE TITLE
ImportError: No module named autoencoder

### ISSUE THREAD

**Issue by isalirezag** [vitruvianscience/OpenDeep#26]
there is not any opendeep.models.single_layer.autoencoder to import DenoisingAutoencoder from :/

**Comment by mbeissinger** [comment]
Ah sorry in a rewrite! I'm working on it, since DenoisingAutoencoder was a specialized version of a Generative Stochastic Network (GSN)

**Comment by isalirezag** [comment]
I appreciate it if it can be fixed soon :)

**Comment by mbeissinger** [comment]
Yep! In the meantime you can see a DenoisingAutoencoder example written here: https://github.com/vitruvianscience/OpenDeep/issues/27
